storage/StorageExampleSwiftUI/ImagePicker.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  ImagePicker.swift
+//  StorageExampleSwiftUI
+//
+//  Created by Ben Reed on 3/25/21.
+//  Copyright © 2021 Google Inc. All rights reserved.
+//
+
+import SwiftUI
+import Firebase
+
+struct ImagePickerRepresentable {
```

Comment:
nice job with what you have going on this file. 👏  hopefully Apple provides us with a standard `PhotoPicker` view that saves us from having to do all of this 🤞

Reply:
Thank you. Maybe there will be something announced in June. It really seems like SwiftUI was a second class citizen for the new photo APIs, so I expect them to fix that.